custom-metrics-stackdriver-adapter/glide.lock

```diff
@@ -0,0 +1,494 @@
+hash: 807f261d8170be02e0c3802e4d005c334a04419a8b2e5f75d68498385202f5ff
```

Comment:
if you actually want to use glide, you should keep the `glide.lock` file -- it "pins" the full dependency set and specific versions so that it doesn't have to be recalculated every time.  If you're not using glide, it's fine to remove all glide-related files.